offa.csl

```diff
@@ -0,0 +1,430 @@
+        <date date-parts="year" form="text" variable="issued" prefix=" (" suffix=")"/>
+      </if>
+    </choose>
+    <text variable="page" prefix=" "/>
+    <group>
+      <choose>
+        <if match="any" variable="title">
+          <text term="sub verbo" form="short" prefix=" " suffix=" "/>
+          <text variable="title"/>
+        </if>
+      </choose>
+    </group>
+    <text variable="edition" prefix=" [" suffix="]"/>
+    <choose>
+      <if match="any" variable="author">
+        <group prefix=" (" suffix=")">
+          <names variable="author"/>
+        </group>
+      </if>
+    </choose>
+  </macro>
+  <macro name="webpage-date">
+    <choose>
+      <if type="webpage" match="any">
+        <choose>
+          <if match="none" is-uncertain-date="issued">
+            <date date-parts="year" form="text" variable="issued" prefix=", "/>
+          </if>
+        </choose>
+      </if>
+    </choose>
+  </macro>
+  <citation et-al-min="3" et-al-use-first="1" disambiguate-add-givenname="true" givenname-disambiguation-rule="all-names-with-initials" disambiguate-add-year-suffix="true">
+    <sort>
+      <key variable="issued"/>
+      <key macro="contributors-short-citation"/>
+    </sort>
+    <layout prefix="(" suffix=")" delimiter="; ">
+      <group delimiter=", ">
+        <group delimiter=" ">
+          <text macro="contributors-short-citation"/>
+          <text macro="date"/>
+        </group>
+        <text variable="locator"/>
+      </group>
+    </layout>
+  </citation>
+  <bibliography hanging-indent="false" line-spacing="1" entry-spacing="1">
+    <sort>
+      <key macro="contributors-short-biblio"/>
+      <key variable="issued"/>
+    </sort>
+    <layout>
+      <group suffix=".">
+        <group suffix=": ">
+          <text macro="contributors-short-biblio" suffix=" "/>
+          <text macro="date"/>
+        </group>
+        <group>
+          <choose>
+            <if type="entry-encyclopedia entry-dictionary" match="any">
+              <text macro="encyclopedia-article"/>
+            </if>
+            <else>
+              <text macro="contributors"/>
+              <text macro="title" strip-periods="false"/>
+              <text macro="review"/>
+              <text macro="short-container-contributors"/>
+              <text macro="locators"/>
+              <text macro="issue"/>
+              <text macro="series"/>
+              <choose>
+                <if type="article article-journal article-magazine article-newspaper" match="none">
+                  <text macro="translator" suffix=" "/>
+                </if>
+              </choose>
+              <text macro="webpage-date"/>
+              <text macro="place-date"/>
+              <text macro="journal-date"/>
+              <text macro="locators-chapter"/>
+              <text macro="locators-article"/>
+            </else>
+          </choose>
+          <choose>
+              <text macro="doi"/>
+            </if>
```

Comment:
```suggestion
          <choose>
          <if variable="DOI" match="any">
              <text macro="doi"/>
            </if>
```

sorry, you do need this here because of the else for the URL